.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -6,29 +6,27 @@ body:
     id: help
     attributes:
       value: |
-        > **Note**
-        > GitHub Issues use [GitHub Flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for formatting.
+        > Titles should be short, descriptive, and compelling.
   - type: textarea
     id: issue
     attributes:
-      label: Issue Description
+      label: Issue description and context
       description: |
-        Describe the issue you are adding or content you are suggesting.
-        Share any next steps that should be taken our outcomes that would be beneficial.
+        Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax – links and images welcome! Share desired outcomes or potential next steps.
     validations:
       required: true
   - type: textarea
-    id: additional-context
+    id: acceptance-criteria
     attributes:
-      label: Additional Context (optional)
-      description: "Include additional references (screenshots, design links, documentation, etc.) that are relevant"
+      label: Acceptance criteria
+      description: "If known, share 1-3 statements that would need to be true for this issue to be considered resolved. Use a checklist."
   - type: textarea
     id: issue-links
     attributes:
-      label: Issue Links
+      label: Other issues
       description: |
-        What other issues does this story relate to and how?
+        Add the issue number of other issues this relates to and how.
         
         Example:
-        - 🚧 Blocked by: #123
-        - 🔄 Relates to: #234
\ No newline at end of file
+        - 🚧 Blocks/is blocked by #123
```

Comment:
Blocks and Blocked by are two different things. They should have their own lines. Also, we should remove the emoji. it makes copy and pasting a nightmare. I reviewed both the default suggested github syntax as well as syntax used by [Zenhub](https://help.zenhub.com/support/solutions/articles/43000010349-creating-github-issue-dependencies-in-zenhub) and [Codetree](https://codetree.com) in the event we ever utilize extensions to our Github Org
Suggested update:
Blocked by 
Blocking 
Relates to

Reply:
* `blocks` and `is blocked by` are certainly different, though related, things. 😄 That's what the `/` is for. I'll put them on different lines if folks prefer.
* Since the emoji are in description text, I like the emojis because they provide a quick visual that supports the words. Do we need to optimize for people copying/pasting the words "Blocks" or "Relates to"?

Reply:
Ended up removing the emoji and using the `placeholder` attribute